bitnami/odoo/templates/ingress.yaml

```diff
@@ -32,6 +32,11 @@ spec:
             pathType: {{ .Values.ingress.pathType }}
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
+          - path: /longpolling/
```

Comment:
What if Odoo is not intended to be accessed on `/` but on `/odoo`? I imagine the intended path would be `/odoo/longpolling`, right?